build-data-capturing-gradle-samples/README.md

```diff
@@ -0,0 +1,97 @@
+# Build Data Capturing Gradle Samples
```

Comment:
Should we make any updates to the script header comments to better align with what I've written here? (e.g. [this](https://github.com/gradle/gradle-enterprise-build-config-samples/blob/df467f4321aba0ebe95b10fff90eba62616053d5/build-data-capturing-gradle-samples/capture-dependency-resolution/gradle-dependency-resolution.gradle#L2-L7))

Should we remove the part of the comment that mentions how to "apply" the script?

Reply:
IMO: let's remove the direction on how to apply the scripts.

@etiennestuder do you agree we should remove the script header comments on what they do in favor of this readme?

Reply:
I've removed all occurrences of how to apply the scripts from the scripts themselves in fbfcf3e2fc0e09d139c7a18c45491b989bb31255.

Note: There is still a description in each file.